addons/mail/static/src/models/message_action_view.js

```diff
@@ -0,0 +1,216 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { clear, insertAndReplace } from '@mail/model/model_field_command';
+import { markEventHandled } from '@mail/utils/utils';
+
+registerModel({
+    name: 'MessageActionView',
+    recordMethods: {
+        /**
+         * @private
+         * @param {MouseEvent} ev
+         */
+        onClick(ev) {
+            switch (this.messageAction.messageActionListOwner) {
+                case this.messageAction.messageActionListOwnerAsDelete:
+                    this.update({ deleteConfirmDialog: insertAndReplace() });
+                    break;
+                case this.messageAction.messageActionListOwnerAsEdit:
+                    this.messageAction.messageActionListOwner.messageView.startEditing();
+                    break;
+                case this.messageAction.messageActionListOwnerAsMarkAsRead:
+                    this.messageAction.messageActionListOwner.message.markAsRead();
+                    break;
+                case this.messageAction.messageActionListOwnerAsReaction:
+                    if (!this.reactionPopoverView) {
+                        this.update({ reactionPopoverView: insertAndReplace() });
+                    } else {
+                        this.update({ reactionPopoverView: clear() });
+                    }
+                    break;
+                case this.messageAction.messageActionListOwnerAsReplyTo:
+                    markEventHandled(ev, 'MessageActionList.replyTo');
+                    this.messageAction.messageActionListOwner.messageView.replyTo();
+                    break;
+                case this.messageAction.messageActionListOwnerAsToggleCompact:
+                    this.messageAction.messageActionListOwner.update({ isCompact: !this.messageAction.messageActionListOwner.isCompact });
+                    break;
+                case this.messageAction.messageActionListOwnerAsToggleStar:
+                    this.messageAction.messageActionListOwner.message.toggleStar();
+                    break;
+                default:
+                    break;
```

Comment:
isn't that useless?

Reply:
It's functionally useless but it's personal code styling to do that since it is suggested to do so as said, 
`As a matter of good form, put a break after the last case (the default here) even though it's logically unnecessary. `
But if it is better to avoid using that, it's fine for me to remove it!

Reply:
I'm not talking about the break specifically but also the empty default in the first place.